examples/markov_text_generation.c

```diff
@@ -1,10 +1,13 @@
 #define CGL_LOGGING_ENABLED
 #define CGL_EXCLUDE_WINDOW_API
 #define CGL_EXCLUDE_NETWORKING
-#define CGL_EXCLUDE_GRAPHICS_API
+// Bloom requires Graphics
```

Comment:
This markov text example doesn't really use Bloom so is it not better to exclude graphics so that this can be compiled without linking OpenGL